source/d_gi.cpp

```diff
@@ -1185,6 +1185,7 @@ static gamemodeinfo_t giDoomSW =
    DOOMDEFSOUND,      // defSoundName
    doom_skindefs,     // skinSounds
    doom_soundnums,    // playerSounds
+   NULL,              // titleMusName
```

Comment:
Please replace instances of NULL with nullptr. Most NULLs in EE that're assigned to pointers will be that way because it's legacy code from C.